packages/node/rollup.config.mjs

```diff
@@ -0,0 +1,11 @@
+// The following is only possible from Node 18 onwards
+// import pkg from "./package.json" assert { type: "json" };
+
+// Until we only support Node 18+, this should be used instead
+// (see https://rollupjs.org/guide/en/#importing-packagejson)
+import { createRequire } from "node:module";
+import createConfig from '../../rollup.common.mjs';
+const require = createRequire(import.meta.url);
+const pkg = require("./package.json");
+
+export default createConfig(pkg);
```

Comment:
All the dependencies of the package need to be added to the `external` array, so that they are not bundled with the source code, and rather resolved as transitive dependencies by the consuming app.

Reply:
https://github.com/inrupt/solid-client-authn-js/pull/3022/commits/2cc1c08c6adc63272711070ca85da40a68343b12 now gets them directly from the package.json dependencies